frontend/src/components/common/StatusPill.vue

```diff
@@ -0,0 +1,61 @@
+<script setup lang="ts">
+import { toKebabCase } from '@/utils/utils';
+
+// Props
+const { authStatus } = defineProps<{
+  authStatus?: string;
+}>();
+</script>
+
+<template>
+  <div class="col-1">
+    <div :class="['authIndicator', toKebabCase(authStatus)]">
+      {{ authStatus }}
+    </div>
+  </div>
+</template>
+<style scoped lang="scss">
+.authIndicator {
```

Comment:
Style: IIRC best practices for CSS element naming should follow BEM naming conventions. In this case, this class should be kebab cased to `.auth-indicator`.
- Ref: https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/

Reply:
done, thank you!